include/mender-deployment-data.h

```diff
@@ -89,6 +92,14 @@ mender_err_t mender_create_deployment_data(
  */
 mender_err_t mender_deployment_data_add_payload_type(mender_deployment_data_t *deployment_data, const char *payload_type);
 
+/**
+ * @brief Get payload type
+ * @param deployment_data Deployment data
+ * @param payload_type Payload type
+ * @return MENDER_OK on success, otherwise MENDER_FAIL
```

Comment:
I would add here a `@note` -> Because we don't support Artifacts with more than one payload, it will return always the first payload type from the list.